service-request-infno-spec-1.0.0-swagger.json

```diff
@@ -0,0 +1,213 @@
+{
+  "openapi" : "3.0.0",
+  "servers" : [ {
+    "url" : "https://qaeservices1.capetown.gov.za/coct/api"
+  } ],
+  "info" : {
+    "description" : "This is a example API for Service Request.",
+    "version" : "1.0.0",
+    "title" : "Service Request Info API",
+    "license" : {
+      "name" : "Apache 2.0",
+      "url" : "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "tags" : [ {
+    "name" : "3rd_parties"
+  }, {
+    "name" : "media"
+  } ],
+  "paths" : {
+    "/service-request/status/" : {
+      "get" : {
+        "tags" : [ "3rd_parties" ],
+        "summary" : "Get service request API Info",
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "service_request_no",
+          "description" : "maximum number of records to return",
+          "required" : true,
+          "schema" : {
+            "type" : "integer",
+            "format" : "int32",
+            "minimum" : 0,
+            "maximum" : 9
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "OK",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "array",
```

Comment:
I think we would only ever return one `ServiceRequestSchema` from this path

Reply:
Please elaborate, I don't think I fully understand

Reply:
so the return type of this path can just be:

```json
"content" : {
  "application/json" : {
    "schema" : {
      "$ref":  "#/components/schemas/ServiceRequestSchema"
    }
  }
}
```